docs/quant/pyquant/ssvi.py

```diff
@@ -0,0 +1,238 @@
+        if not (theta >= 0):
+            raise ValueError("Theta not >= 0")
+        self.theta = theta
+
+
+@nb.experimental.jitclass([("zeta", nb.float64)])
+class Zeta:
+    def __init__(self, zeta: nb.float64):
+        if not (zeta > 0):
+            raise ValueError("Zeta not > 0")
+        self.zeta = zeta
+
+
+@nb.experimental.jitclass([("lambda_", nb.float64)])
+class Lambda:
+    def __init__(self, lambda_: nb.float64):
+        if not (lambda_ >= 0):
+            raise ValueError("Lambda not >= 0")
+        self.lambda_ = lambda_
+
+
+@nb.experimental.jitclass([("eta", nb.float64)])
+class Eta:
+    def __init__(self, eta: nb.float64):
+        if not (eta >= 0):
+            raise ValueError("Eta not >= 0")
+        self.eta = eta
+
+
+@nb.experimental.jitclass([("beta", nb.float64)])
+class Beta:
+    def __init__(self, beta: nb.float64):
+        if not (beta >= 0):
+            raise ValueError("Beta not >= 0")
+        self.beta = beta
+
+
+@nb.experimental.jitclass([("alpha", nb.float64)])
+class Alpha:
+    def __init__(self, alpha: nb.float64):
+        self.alpha = alpha
+
+
+@nb.experimental.jitclass([("gamma_", nb.float64)])
+class Gamma_:
+    def __init__(self, gamma_: nb.float64):
+        self.gamma_ = gamma_
+
+
+@nb.experimental.jitclass(
+    [
+        ("delta_param", nb.float64),
+        ("mu", nb.float64),
+        ("rho", nb.float64),
+        ("theta", nb.float64),
+        ("zeta", nb.float64),
+        ("lambda_", nb.float64),
+        ("eta", nb.float64),
+        ("beta", nb.float64),
+        ("alpha", nb.float64),
+        ("gamma_", nb.float64),
+    ]
+)
+class SVINaturalParams:
+    def __init__(
+        self, delta_param: DeltaParam, mu: Mu, rho: Rho, theta: Theta, zeta: Zeta
+    ):
+        self.delta_param = delta_param.delta_param
+        self.mu = mu.mu
+        self.rho = rho.rho
+        self.theta = theta.theta
+        self.zeta = zeta.zeta
+
+    def array(self) -> nb.float64[:]:
+        return np.array([self.delta_param, self.mu, self.rho, self.theta, self.zeta])
+
+
+class SSVI:
+    def __init__(
+        self,
+        vol_smile_chain_spaces: list[VolSmileChainSpace],
+        is_log: bool = False,
+    ) -> None:
+        self.is_log = is_log
+        self.raw_params_list = []
+        self.natural_params_list = []
+        self.vol_smile_chain_spaces = vol_smile_chain_spaces
+
+    def calibrate(
+        self,
+        for_delta_space: bool = False,
+    ) -> None:
+        for vol_smile_chain_space in self.vol_smile_chain_spaces:
+            if self.is_log:
+                print("\n")
+                print(
+                    f"======== Get natural params for tau = {vol_smile_chain_space.T} ======== "
+                )
+                print(f"Market IV {vol_smile_chain_space.sigmas}")
+            # 1. for every time to maturity calibrate it's own SVI with raw params
+            svi_calc = SVICalc()
+
+            svi_calibrated_params, svi_error = svi_calc.calibrate(
```

Comment:
you need an SVI `calibrate` method that calibrates only `b`, `rho` and `sigma`, with `a,mu=0` to delta space quotes.